injector/disk_failure.go

```diff
@@ -101,7 +101,13 @@ func (i *DiskFailureInjector) Inject() error {
 		exitCode = i.spec.OpenatSyscall.GetExitCodeInt()
 	}
 
-	return i.config.Cmd.Run(pid, i.spec.Path, exitCode)
+	for _, path := range i.spec.Paths {
+		if err := i.config.Cmd.Run(pid, path, exitCode); err != nil {
```

Comment:
You may want to have a look at the BackgroundCmd struct [here](https://github.com/DataDog/chaos-controller/blob/323596d971a465428ac8aa5180e7a77f900af0aa/command/command.go#L112)

Used [here](https://github.com/DataDog/chaos-controller/blob/323596d971a465428ac8aa5180e7a77f900af0aa/injector/dns_disruption.go#L130)

It's not directly part to this review though.

A comment may be necessary as I had to check the implementation to confirm it was not a synchronous call.

Reply:
Interesting 👍 I will take a look!